sycl/test/native_cpu/call_host_func.cpp

```diff
@@ -0,0 +1,26 @@
+// REQUIRES: native_cpu_be
+// RUN: %clangxx -fsycl -fsycl-targets=native_cpu %s -o %t
+// RUN: env ONEAPI_DEVICE_SELECTOR=native_cpu:cpu %t
+// This test is needed since we need to make sure that there no
+// "multiple definitions" linker errors when a function appears
+// both in the host and in the device module.
+#include <CL/sycl.hpp>
```

Comment:
```suggestion
#include <sycl/sycl.hpp>
```
We should be using the SYCL 2020 include path. Likewise, tests using `cl::sycl::` should just use `sycl::`.

Reply:
Done, thank you for pointing it out